docs/docs/identity-providers.md

```diff
@@ -105,20 +105,20 @@ Navigate to **User Settings** then **Applications** using the left-hand menu.
 On the **Applications** page, add a new application by setting the following parameters:
 
 Field        | Description
------------- | --------------------------------------------
+------------ | --------------------------------------------------------------------
 Name         | The name of your web app
-Redirect URI | `https://${redirect-url}/oauth2/callback`
+Redirect URI | [Redirect URL] (e.g.`https://auth.corp.example.com/oauth2/callback`)
 Scopes       | **Must** select **read_user** and **openid**
 
 ![Create New Credentials](./gitlab/gitlab-create-application.png)
 
 1.Click **Save Application** to proceed.
 
-Your `Client ID` and `Client Secret` will be displayed:
+Your [Client ID] and [Client Secret] will be displayed:
 
-![Gitlab OAuth Client ID and Secret](./gitlab/gitlab-credentials.png)
+![Gitlab OAuth [Client ID] and Secret](./gitlab/gitlab-credentials.png)
```

Comment:
The search and replace also doubled the brackets up inside the alt text, here and in a few other places in this file.

Reply:
Hopefully that did the trick. I've pushed an update to the site and a new git commit. Thanks @mbarrien  👍